### DISCUSSION TITLE
default keybind for "edit current path"

### DISCUSSION THREAD

**Discussion by goyalyashpal** [doublecmd/doublecmd#796] [Suggestions]
To go to address bar and edit it, the keybind is:
* In windows explorer and IE browser: <tt>[Alt+D]</tt> (it is currently not taken in dc)
* In other modern browsers, the pro'lly more known keybind is <tt>[Ctrl+L]</tt> (but that's occupied in dc)

is there a default or de-facto keybind for going to address bar or "edit current path" on the file explorers on linux distros too?
* if yes, then maybe those can be set as default in dc too?

Re: Any way to directly invoke "edit current path (as [cm_EditPath](https://doublecmd.github.io/doc/en/cmds.html#cm_EditPath))"
\- https://github.com/doublecmd/doublecmd/issues/792#issuecomment-1417348919

[Alt+D]: https://github.com/msys2/MSYS2-packages/issues/2744#issuecomment-1009369509 "by @cbrt4"
[Ctrl+L]: https://github.com/doublecmd/doublecmd/issues/793#issuecomment-1420148183 "by @PetbkA"

**Comment by Skif-off** [comment]
We can use Alt+Letter in the following cases:
- accelerators ([see "mnemonics "](https://en.wikipedia.org/wiki/Keyboard_shortcut));
- quick search/filter ([see "way to call"](https://doublecmd.github.io/doc/en/help.html#cm_QuickSearch)).

I tried: run DC > open Options window > open the "Hot keys" ection > find `cm_EditPath` > set Alt+D > Apply > Ok = ~20 seconds.

P.S. If nothing is selected and the cursor is on the first item (".."), F2 and Shift+F6 will call `cm_EditPath`.

**Reply by goyalyashpal** [reply]
> the cursor is on the first item (".."), F2

hi! thanks for input. this is a nice workaround, but it fails where there's no `..` i.e. no parent directory, for example when u are in root location (say, a drive) already.

**Reply by avmaksimov** [reply]
> > the cursor is on the first item (".."), F2
> 
> hi! thanks for input. this is a nice workaround, but it fails where there's no `..` i.e. no parent directory, for example when u are in root location (say, a drive) already.

Do you remove Shift+F6 for command cm_RenameOnly? If you do that all will work good!

**Reply by avmaksimov** [reply]
But I agree Shift+F6 for cm_EditPath is better than for cm_RenameOnly, which has already F2.

**Alex2000**, what do you think?

**Reply by alexx2000** [reply]
`Shift+F6` is a standard hotkey for `cm_RenameOnly` (like in TC). We should use some other hotkey for `cm_EditPath`.
